modules/oe_content_tender/oe_content_tender.info.yml

```diff
@@ -0,0 +1,30 @@
+name: OpenEuropa Tender Content
+description: Contains the EC corporate Call for tender content type.
+package: OpenEuropa
+type: module
+core: 8.x
+
+dependencies:
```

Comment:
List of dependencies has to be:
```
  - composite_reference:composite_reference
  - drupal:datetime
  - drupal:link
  - oe_content:oe_content
  - oe_content:oe_content_departments_field
  - oe_content:oe_content_reference_code_field
  - oe_content:oe_content_publication
  - oe_media:oe_media
```

Reply:
Or instead of  - oe_content:oe_content_publication has to be oe_content:oe_content_documents_field.

Reply:
Or instead of - oe_content:oe_content_publication has to be oe_content:oe_content_documents_field from EPIC-Project.

Reply:
You are right @yenyasinn, let's remove this field all at once and create a follow up,  @nagyad will do that. We will add it when project will be merged, as discussed.